src/nile/core/init.py

```diff
@@ -1,30 +1,11 @@
 """Command to kickstart a Nile project."""
 import logging
-import subprocess
-import sys
 from distutils.dir_util import copy_tree
 from pathlib import Path
 
-from nile.core.install import install
-
 
 def init():
     """Kickstart a new Nile project."""
-    # install cairo dependencies
-    subprocess.check_call(
-        [sys.executable, "-m", "pip", "install", "ecdsa", "fastecdsa", "sympy"]
-    )
```

Comment:
are these installed by `cairo-lang` now? it wasn't the case before

Reply:
Yes, indeed